src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/totrinnskontroll/AngreSendTilBeslutterService.kt

```diff
@@ -77,11 +77,31 @@ class AngreSendTilBeslutterService(
 
     private fun validerKanAngreSendTilBeslutter(saksbehandling: Saksbehandling) {
         validerSaksbehandlerSomAngrer(saksbehandling)
+
+        validerGodkjenneVedtakOppgaveIkkeErPlukket(saksbehandling)
```

Comment:
Dette gjøres vel allerede i `validerOppgave` ?

Reply:
For meg virker det som at det fantes 2 oppgaver av typen `GodkjenneVedtak` som ikke var ferdigstilt i databasen

Reply:
Feilet er ju at det finnes 2 behandle-sak-oppgaver, når man sender til beslutter på nytt. Dette fordi han trigget angre før den første Behandle-sak-oppgaven er ferdigstilt

For å håndtere det burde man kanskje ha:
* Send til beslutter sjekker at det ikke finnes en "GodkjenneVedtak/BehandleUnderkjentVedtak" som ikke er ferdigstilt
* Angre/totrinnskontroll sjekker at det ikke en "GodkjenneVedtak" som ikke er ferdigstilt

Reply:
Oi, det var der jo :/